packages/flutter_tools/lib/executable.dart

```diff
@@ -179,8 +179,11 @@ Future<Null> _exit(int code) async {
   logger.flush();
 
   // Give the task / timer queue one cycle through before we hard exit.
-  await Timer.run(() {
+  Completer<Null> completer = new Completer<Null>();
+  Timer.run(() {
```

Comment:
That wouldn't wait til the end of the timer though, would it? I guess I don't know what the original await was trying to do. In practice it wasn't doing anything and I could just remove it with no change of semantics...